app/src/main/java/no/usn/mob3000/ui/screens/user/ProfileEditScreen.kt

```diff
@@ -109,6 +112,7 @@ fun ProfileEditScreen(
                     onValueChange = { selectedCountry = it },
                     readOnly = true,
                     label = { Text(stringResource(R.string.profile_edit_nationality)) },
+                    textStyle = TextStyle(color = MaterialTheme.colorScheme.onBackground),
```

Comment:
[See this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847123849).